### PR TITLE
Replace remaining $a->page by DI::page()

### PR DESCRIPTION
**Pull request by MrPetovan**
Follow-up to #8035

Look what I found.

### CODE REVIEW

**Comment by nupplaphil** [comment]
-.-* ... O dear, O dear ... My strength is really more conceptional than operative -.-* .. shame on me ..

**Comment by MrPetovan** [comment]
It's fine, you do the thinkin', we do the workin'.

File: src/Module/Contact.php [hunk 1]
```diff
@@ -947,7 +947,7 @@ private static function getConversationsHMTL($a, $contact_id, $update)
 		}
 
 		if (DBA::isResult($contact)) {
-			$a->page['aside'] = '';
+			DI::page()['aside'] = '';
```
Comment (annando) [inline src/Module/Contact.php]: Is this working and valid?

Comment (MrPetovan) [reply src/Module/Contact.php]: Yes, the `Page` class implements the `ArrayAccess` interface which allows array-style access to class members.

Comment (annando) [reply src/Module/Contact.php]: I really would prefer dedicated methods for that :-)

Comment (MrPetovan) [reply src/Module/Contact.php]: Yep, me too, but it can come in later.

**Pull request merged**